core/src/scene/sceneLoader.cpp

```diff
@@ -1160,14 +1158,15 @@ Filter SceneLoader::generateNoneFilter(Node _filter, Scene& scene) {
 }
 
 void SceneLoader::parseStyleParams(Node params, Scene& scene, const std::string& prefix,
-                                   std::vector<StyleParam>& out) {
+                                   std::vector<StyleParam>& out, bool unified) {
 
     for (const auto& prop : params) {
+        unified |= prefix == "text";
         std::string key;
```

Comment:
What happens when an icon has an empty ```text``` block? The unified property would need to be propagated up. Or one adds some default text property so that it can be identified as icon with text